### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by DianaBeki**
In this task, I did the following
- Build  works section
- Build about section
- Checked for linter error
- Styled the sections to match the design
-

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,5 +45,239 @@ <h3 class="connect">LET’S CONNECT</h3>
       </div>
       </div>
   </section>
+  
+ <!--Works Section  -->
+ <section class="works">
+  <div class="snap-card">
```
Comment (vickymarz) [inline index.html:51]: - [ ] KIndly reduce the spacing between the content of your project cards to match your Figma design 👍🏽 

Figma Design 👇🏽 
image

Your Design 👇🏽 
image

File: index.html [hunk 2]
```diff
@@ -44,5 +45,239 @@ <h3 class="connect">LET’S CONNECT</h3>
...
+    </div>
+  </div>
+</section>
+
+<!-- About myself Section -->
+
+<section class="bio">
+  <div class="primary-info">
+    <div class="title">
+      <h1>About Myself</h1>
```
Comment (vickymarz) [inline index.html:203]: - [ ] I will strongly recommend you wrap your `about me` title in a heading tag apart from `<h1>` as you should only have a single `<h1>` tag in your HTML file. You can use `<h2>` tag

e.g `<h2>About Myself</h2>`

Comment (vickymarz) [inline index.html:203]: - [ ] Kindly increase the font weight of your about me title to match the figma design

FIgma Design 👇🏽 
image

Your Design 👇🏽 
image

File: index.html [hunk 3]
```diff
@@ -44,5 +45,239 @@ <h3 class="connect">LET’S CONNECT</h3>
...
+    </div>
+    <div class="description">
+      <p>
+        I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don’t hestiate to
+        contact me.
+      </p>
+    </div>
+    <div class="social_media"><a href="#">LET’S CONNECT</a></div>
```
Comment (vickymarz) [inline index.html:213]: - [ ] Kindly ensure that the color of your `LET’S CONNECT` matches the figma design

Figma Design 👇🏽 
image

Your Design 👇🏽 
image

File: index.html [hunk 4]
```diff
@@ -44,5 +45,239 @@ <h3 class="connect">LET’S CONNECT</h3>
...
+            <img src="./images/html.png" alt="" />
+          </div>
+          <h4>HTML</h4>
+        </li>
+        <li class="lang">
+          <div class="langImg">
+            <img src="images/css.png" alt="" />
+          </div>
+          <h4>CSS</h4>
+        </li>
```
Comment (vickymarz) [inline index.html:267]: - [ ] Kindly reduce the font-weight of your list of technologies to match the Figma design 👍🏽

Figma Design 👇🏽 
image

Your Design 👇🏽 
image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @DianaBeki, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DianaBeki** [comment]
Adjusted the following

- Reduced  spacing in the content cards
- Changed font weight for about myself section
- Changed the let's connect text color to match the figma design
- Changed font weight for the list of technologies

**Review by ichala — APPROVED** [review]
Hi @DianaBeki  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**